tools/PI/DevHome.PI/BarWindow.xaml.cs

```diff
@@ -292,7 +294,7 @@ private void CreateMenuItemFromTool(ExternalTool item)
 
     private void ExternalToolItem_PropertyChanged(object? sender, PropertyChangedEventArgs e)
     {
-        if (sender is ExternalTool item && e.PropertyName == nameof(ExternalTool.MenuIcon))

```

Comment:
>== [](http://example.com/codeflow?start=58&length=2)

We do this sort of comparison throughout a lot of the code. Should we be doing string.equals instead?

Reply:
So it seems, and yes I fixed this one in the 2nd commit.